tests/test_radius_global.rb

```diff
@@ -65,11 +65,13 @@ def test_radius_global
     assert_equal(global.timeout,
                  2)
 
-    global.key_set('44444444', nil)
-    assert_equal(Cisco::RadiusGlobal.radius_global[id].key,
-                 '44444444')
-    assert_equal(global.key,
-                 '44444444')
+    if platform == :nexus
```

Comment:
What's the reason behind skipping the setter test on XR?

I'm also not sure why the setter is named ``key_set`` instead of ``key=`` (it wasn't done in this PR, I know).